browser/components/MarkdownPreview.js

```diff
@@ -127,13 +127,21 @@ h1, h2 {
   border: none;
 }
 
+h3 {
+  margin: 1em 0 0.4em;
+}
+
+h4, h5, h6 {
+  margin: 1em 0 0.5em;
+}
+
 h1 {
-  padding-bottom: 4px;
+  padding: 0.2em 0 0.2em;
   margin: 1em 0 8px;
 }
 
 h2 {
-  padding-bottom: 0.2em;
+  padding: 0.2em 0 0.2em;
   margin: 1em 0 0.37em;
```

Comment:
Can you change `margin` into `1em 0 0.7em`? otherwise the `h2` will make the line disappear at 80%
```suggestion
  margin: 1em 0 0.7em;
```